src/middleware.ts

```diff
@@ -10,9 +10,8 @@ export const middleware = async (request: NextRequest) => {
   const unauthenticatedOnlyPrefixes = ["/login"] as const;
 
   if (adminOnlyPrefixes.some((path) => pathname.startsWith(path))) {
-    const token = request.cookies.get("token");
-    const userInfo = await getUser(token?.value);
-    if (!userInfo.ok || userInfo.user.status !== "Admin") {
+    const userInfo = await verifyCodefunWithMemo();
```

Comment:
Why? Is `cookies()` intended for use in middleware?

Reply:
oops, my bad